Bluepill-cli/bluepill-cli-release.xcconfig

```diff
@@ -7,5 +7,4 @@
 //  distributed under the License is distributed on an "AS IS" BASIS,
 //  WITHOUT WARRANTIES OF ANY KIND, either express or implied.  See the License for the specific language governing permissions and limitations under the License.
 
-#include "../Configurations/common.xcconfig"
```

Comment:
when is this config used?

Reply:
bluepill-cli needs to link the frameworks.